server/testutils/factories/cas1SpaceBooking.ts

```diff
@@ -8,6 +8,7 @@ import userFactory from './user'
 
 import staffMemberFactory from './staffMember'
 import { DateFormats } from '../../utils/dateUtils'
+import cas1SpaceBookingNonArrivalFactory from './cas1SpaceBookingNonArrival'
 
 const addTime = (date: Date): Date => new Date(date.getTime() + faker.number.int({ max: 60 * 60 * 24 * 1000 }))
```

Comment:
I still don't think this is needed -- the faker date methods always return full dateTime objects...